### ISSUE TITLE
Canadian build host=mingw32, no toolchain NLS: gettext is built for host

### ISSUE THREAD

**Issue by tvb377** [crosstool-ng/crosstool-ng#699]
Canadian cross usage: build=Linux, host=mingw32, target=x
Selecting glibc for target leads to GETTEXT_NEEDED.
Selecting NLS for the toolchaing would also result in GETTEXT_NEEDED.

However, ct-ng HEAD of 11 Mar 2017 builds gettext for host even if toolchain NLS support is deselected.
This should not be necessary.

**Comment by tvb377** [comment]
@stilor said in an email:
I see.
The commit message states:

commit 9e81836b8124efd11805e8050034492a8831208b
Author: Ray Donnelly <mingw.android@gmail.com>
Date:   Sat Oct 24 01:49:56 2015 +0100

     Add gettext and libiconv as companion libs

     .. they're needed for the RPC generation in glibc
     on both Cygwin and MinGW-w64.

So apparently, the cross_rpcgen (on *build*, rather than *host*) needs it.
In case of a simple cross, of course, build==host.

**Comment by stilor** [comment]
I assume you submitted this as a follow-up to the discussion we had on the crossgcc mailing list. If so, can you attach the gettext patch you submitted?

AFAIU, there are two issues here:
- gettext may be unnecessarily built for the *host*, even if it is only needed for the *build*
- if gettext needs to be built for mingw *host*, it fails to build

**Comment by tvb377** [comment]
Yes, this issue is for your first bullet point, issue #702 is for the second.
